src/dune_rules/glob_files.ml

```diff
@@ -0,0 +1,75 @@
+open! Import
+
+type t =
+  { glob : String_with_vars.t
+  ; recursive : bool
+  }
+
+module Path_split = struct
+  (* A source path, or a build path which has been split into its _build/foo
```

Comment:
```suggestion
  (* A source path, or a build path, which has been split into its _build/foo
```
Either a comma after build path or no comma after source path.

Reply:
I've reworded it slightly to clarify that the path is only split in the case of a build path.

Reply:
Great!